src/styles/_catalog.scss

```diff
@@ -1,22 +1,30 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  padding: 10px 20px;
+  display: grid;
+  justify-content: center;
+  gap: 48px;
 
```

Comment:
"There should be one column below 488px" think you missed this requirement :)

just add here `grid-template-columns: repeat(1, 200px);`

Reply:
just thought this limit was not necessary as there was 1 column below 488px without this limit :)